### PR TITLE
Basic Recruit Training

### PR DESCRIPTION
**Pull request by Dillonsd**
Basic recruit training completed.

### CODE REVIEW

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:47]: You should have a new line at the end of a file.

File: Recruit-Training/Basic-Recruit-Training/person/Person.h
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.h:28]: As mentioned in the instructions, there shouldn't be any modification of `Person.h`

**Review by JessieG123 — CHANGES_REQUESTED** [review]

**Review by JessieG123 — CHANGES_REQUESTED** [review]
Please put a description for the pr.

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (Dillonsd) [reply Recruit-Training/Basic-Recruit-Training/person/Person.cpp]: Added the newline

File: Recruit-Training/Basic-Recruit-Training/person/Person.h
Comment (Dillonsd) [reply Recruit-Training/Basic-Recruit-Training/person/Person.h:28]: Reverted the change

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (duan-le) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:14]: You can just make this `age_ = new int;`.

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (duan-le) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:5]: This can be `age_ = new int;`.

**Review by duan-le — CHANGES_REQUESTED** [review]

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:35]: You should use `std::cout`.  It's the proper C++ way.

**Review by JessieG123 — CHANGES_REQUESTED** [review]

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (Dillonsd) [reply Recruit-Training/Basic-Recruit-Training/person/Person.cpp:15]: Does age need to be an array?

Comment (JessieG123) [reply Recruit-Training/Basic-Recruit-Training/person/Person.cpp:15]: From `Person.h`, it shouldn't be.

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:15]: When you update something please also check where it's being use in other places.

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:5]: It's not an array? Like Duan said, it should just be `age_ = new int`

File: Recruit-Training/Basic-Recruit-Training/person/Person.cpp
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.cpp:31]: You're not passing this into a C function. Not sure why you used `c_str()`.

**Review by JessieG123 — CHANGES_REQUESTED** [review]

File: Recruit-Training/Basic-Recruit-Training/pointers/main.cpp
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/pointers/main.cpp:42]: From the instructions at the top of the file, please make sure you call `MySwap()` in `main()` and pass in the pointer `p` and another pointer that points to `y`.

**Review by JessieG123 — CHANGES_REQUESTED** [review]

File: Recruit-Training/Basic-Recruit-Training/pointers/main.cpp
Comment (duan-le) [reply Recruit-Training/Basic-Recruit-Training/pointers/main.cpp:42]: `You will pass two variables in as argument, one will be the pointer p. The other one will be an integer pointer you make that points to the variable y.
In the function MySwap you will swap the values of the two arguments so the values are changed inside main()`. Then you can `std::cout` the values to check.

File: Recruit-Training/Basic-Recruit-Training/pointers/main.cpp
Comment (duan-le) [inline Recruit-Training/Basic-Recruit-Training/pointers/main.cpp:43]: I know this will work but the instruction specified to create an `integer pointer you make that points to the variable y`. Then you can pass that pointer to `MySwap()`.

**Review by duan-le — CHANGES_REQUESTED** [review]

**Review by duan-le — APPROVED** [review]
Everything looks good to me. Please **DO NOT** merge this branch into `master`. Get an approval from another reviewer and then you can close this PR.

**Review by JessieG123 — APPROVED** [review]
Looks good! Awesome job. As Duan mentioned, do not merge it. Instead, close the PR. Let us know if you have any questions.

**Pull request closed without merging**
